src/components/RawSpectroscopyData.tsx

```diff
@@ -0,0 +1,25 @@
+import { ImagePlot, type NDT } from "@diamondlightsource/davidia";
+import Box from "@mui/material/Box";
+import ndarray from "ndarray";
+
+function RawSpectroscopyData() {
+  const image = ndarray(
+    new Int8Array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]),
+    [4, 4],
+  ) as NDT;
```

Comment:
This appears as a black image, am I missing something?

<img width="856" height="298" alt="image" src="https://github.com/user-attachments/assets/6486132d-2007-4a86-a02c-6d33232daffa" />